Sources/SpringAnimator.swift

```diff
@@ -25,7 +25,7 @@ class SpringAnimator: NSObject {
 
     var initialVelocity: CGPoint = .zero {
         didSet {
-            velocity = initialVelocity
+            velocity = -initialVelocity
```

Comment:
That's a big change 😄

Reply:
I don't really understand why this has to be opposite of the pan gestures velocity  😅 should probably look into this at some point